pkg/koordlet/runtimehooks/server/server.go

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2022 The Koordinator Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package server
+
+import (
+	"fmt"
+	"net"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+	"k8s.io/klog/v2"
+
+	runtimeapi "github.com/koordinator-sh/koordinator/apis/runtime/v1alpha1"
+)
+
+type Options struct {
+	Port int
```

Comment:
It is also better to use network/address for configuration, which can easily support tcp/udp/unixsocket, etc.

Reply:
fixed